ablySpec/RestClient.swift

```diff
@@ -44,7 +46,10 @@ class RestClient: QuickSpec {
                     expect(publishTask.error?.code).toEventually(equal(40005), timeout:testTimeout)
                 }
 
-                pending("should accept a token") {
+                it("should accept a token") {
+                    ARTClientOptions.setDefaultEnvironment("sandbox")
+                    defer { ARTClientOptions.setDefaultEnvironment("sandbox") }
```

Comment:
Oh, silly me. Of course, this is meant to restore the default `nil` value. Fixed at b30271d.